tests/pipeline/nodes/model/yolov4/test_yolo.py

```diff
@@ -21,10 +21,36 @@
 import cv2
 import tensorflow as tf
 from unittest import mock, TestCase
+from pathlib import Path
 from peekingduck.pipeline.nodes.model.yolo import Node
 from peekingduck.pipeline.nodes.model.yolov4.yolo_files.detector import Detector
 
 
+# Yolo model has some issue(Windows fatal exception) with pytest that 
+# limits the number of image tested to 2 for windows, no issue with linux (ubuntu)
+# Only for yolo_test, use the test_human_images_yolo and test_no_human_images_yolo
+# For other model's unit test use the test_human_images and test_no_human_images
+# in conftest.py
```

Comment:
Can specify that this only fails on Github actions with Windows server 2019 and 2016, and not locally on Windows 10?

Reply:
Added the additional information that the error only occurs on Github actions with Windows server 2019 and 2016, and not locally on Windows 10